packages/wallet-service/tests/integration.test.ts

```diff
@@ -248,274 +210,3 @@ test('load wallet, and simulate DLQ event', async () => {
     expect.any(Logger),
   );
 }, 60000);
-
-// eslint-disable-next-line jest/prefer-expect-assertions, jest/expect-expect
-test('start wallet and then receive blocks and txs', async () => {
-  /*
-   * create wallet
-   */
-  await createWallet(mysql, walletId, XPUBKEY, AUTH_XPUBKEY, maxGap);
-  await loadWallet({ xpubkey: XPUBKEY, maxGap }, null, null);
-
-  /*
-   * receive a block
-   */
-  await txProcessor.onNewTxEvent(blockEvent);
-  await checkAfterReceivingFirstBlock(true);
-
-  /*
-   * receive second block
-   */
-  await txProcessor.onNewTxEvent(blockEvent2);
-  await checkAfterReceivingSecondBlock(true);
-
-  /*
-   * add transaction that sends block reward to 2 different addresses on same wallet
-   */
-  await txProcessor.onNewTxEvent(txEvent);
-  await checkAfterReceivingTx1(true);
-
-  // txEvent2 uses utxos that are not from the received blocks, so we must add them to the database
-  await addToUtxoTable(mysql, tx2Inputs);
-
-  /*
-   * add transaction that sends block reward to 2 different addresses, one of which is not in this wallet
-   */
-  await txProcessor.onNewTxEvent(txEvent2);
-  await checkAfterReceivingTx2(true);
-}, 60000);
-
-// eslint-disable-next-line jest/prefer-expect-assertions, jest/expect-expect
-test('receive blocks, start wallet and then receive transactions', async () => {
-  /*
-   * receive a block
-   */
-  await txProcessor.onNewTxEvent(blockEvent);
-  await checkAfterReceivingFirstBlock(false);
-
-  /*
-   * receive second block
-   */
-  await txProcessor.onNewTxEvent(blockEvent2);
-  await checkAfterReceivingSecondBlock(false);
-
-  /*
-   * create wallet
-   */
-  await createWallet(mysql, walletId, XPUBKEY, AUTH_XPUBKEY, maxGap);
-  await loadWallet({ xpubkey: XPUBKEY, maxGap }, null, null);
-
-  /*
-   * add transaction that sends block reward to 2 different addresses on same wallet
-   */
-  await txProcessor.onNewTxEvent(txEvent);
-  await checkAfterReceivingTx1(true);
-
-  // txEvent2 uses utxos that are not from the received blocks, so we must add them to the database
-  await addToUtxoTable(mysql, tx2Inputs);
-
-  /*
-   * add transaction that sends block reward to 2 different addresses, one of which is not in this wallet
-   */
-  await txProcessor.onNewTxEvent(txEvent2);
-  await checkAfterReceivingTx2(true);
-}, 35000);
-
-// eslint-disable-next-line jest/prefer-expect-assertions, jest/expect-expect
-test('receive blocks and tx1, start wallet and then receive tx2', async () => {
```

Comment:
We don't have tests for this, need to create tests that use both the lambdas and the daemon